config/webpack.js

```diff
@@ -430,7 +430,7 @@ module.exports = (env) => {
           /extension-dist/,
           /\.map$/,
           // Ignore both the webapp manifest and the d1-manifest files
-          /data\/d1-manifest/,
+          /data\/d1\/manifests/,
```

Comment:
There's also a line in the `CopyWebpackPlugin` config that excludes `missing_sources.json` - can you check that that's still being excluded correctly?